pkg/products/threescale/reconciler.go

```diff
@@ -272,13 +272,14 @@ func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1a
 	}
 
 	phase, err = r.reconcileComponents(ctx, serverClient)
+	logrus.Infof("Phase: %s reconcileComponents", phase)
 	if err != nil || phase != integreatlyv1alpha1.PhaseCompleted {
 		events.HandleError(r.recorder, installation, phase, "Failed to reconcile components", err)
 		return phase, err
 	}
 
 	phase, err = r.reconcilePodPriority(ctx, serverClient, installation)
-	logrus.Infof("Phase: %s reconcileComponents", phase)
+	logrus.Infof("Phase: %s reconcilePodPriority", phase)
 	if err != nil || phase != integreatlyv1alpha1.PhaseCompleted {
 		events.HandleError(r.recorder, installation, phase, "Failed to pod priority", err)
```

Comment:
@Rajagopalan-Ranganathan We're creating an event here when there may not be an error. Can you add a conditional around the event and only create when there is actually an err.

Reply:
Events.HanldeError() creates events. this is just logging the phase to know if it in progress/waitingofrcomponents.

Reply:
Actually, It will only create the error event if Phase == Failed:
https://github.com/integr8ly/integreatly-operator/blob/a8731a670655d373a40728ea738b421f5e7413f4/pkg/resources/events/events.go#L29